contracts/SimpleNftMarketplace.sol

```diff
@@ -92,11 +97,23 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
     return _listings[listingId].buyer == address(0) && _listings[listingId].seller != address(0);
   }
 
-  function isBlacklistedUser(address userAddress) external view returns (bool isBlacklisted) {}
+  function isBlacklistedUser(address userAddress) external view returns (bool isBlacklisted) {
+    if (_isBlacklistedUser(userAddress) == true) {
+      return true;
+    }
+  }
 
-  function isBlacklistedToken(address tokenContract, uint256 tokenId) external view returns (bool isBlacklisted) {}
+  function isBlacklistedToken(address tokenContract, uint256 tokenId) external view returns (bool isBlacklisted) {
+    if (_isBlacklistedToken(tokenContract, tokenId) == true) {
+      return true;
+    }
+  }
 
-  function isSupportedContract(address tokenContract) external view returns (bool isSupported) {}
+  function isSupportedContract(address tokenContract) external view returns (bool isSupported) {
+    if (_isSupportedContract(tokenContract) == true) {
+      return true;
+    }
```

Comment:
_isSupportedContract(tokenContract) already return bool
so remove the if and just add
```solidity
return _isSupportedContract(tokenContract);
```